hack/load-tests/run-load-test.sh

```diff
@@ -327,15 +327,15 @@ function get_result_and_cleanup_metricagent() {
 
 # shellcheck disable=SC2112
 function get_result_and_cleanup_fluentbit() {
-   RECEIVED=$(curl -fs --data-urlencode 'query=round((sum(rate(fluentbit_input_bytes_total{service="telemetry-fluent-bit-metrics", name=~"load-test-.*"}[5m])) / 1024))' localhost:9090/api/v1/query | jq -r '.data.result[] | .value[1]')
+   RECEIVED=$(curl -fs --data-urlencode 'query=round((sum(rate(fluentbit_input_bytes_total{service="telemetry-fluent-bit-metrics", name=~"load-test-.*"}[20m])) / 1024))' localhost:9090/api/v1/query | jq -r '.data.result[] | .value[1]')
```

Comment:
Why do we not use `avg_over_time` here? The query applies the 20m only to the rate function.

Reply:
fluentbit_input_bytes_total and fluentbit_output_proc_bytes_total are counter type metics, if we user avg_over_time basically we will get the result as how this metrics are over time increased in average

Reply:
then maybe apply "max_over_time" to get a scalar value and not a vector?

Reply:
max_over_time will not work properly with counter type metrics

Reply:
The [xx_over_time](https://prometheus.io/docs/prometheus/latest/querying/functions/#aggregation_over_time) function expects a range vector which is the result of the [rate](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) function. Why that should not work, on what do you base that statement?